src/type-layout.md

```diff
@@ -162,7 +163,24 @@ representation will not change the layout of `Inner`.
 Nominal types without a `repr` attribute have the default representation.
 Informally, this representation is also called the `rust` representation.
 
-There are no guarantees of data layout made by this representation.
+There are very few data layout guarantees made by this representation. The only
```

Comment:
We should probably specify that the guarantees here are required for soundness. Perhaps something of the effect:

"The data layout guarantees made by this representation are those required for soundness."?